ui/src/main/scripts/plugins/directives/instance-view-diagram/ts/instanceViewDiagramDirective.ts

```diff
@@ -198,7 +198,7 @@ module InstanceViewDiagram {
           _.each(d.properties, (property) => {
             if (!!property.value) {
               nodeTT += ('<li class=\'tt-prop\'><strong>' + property.name + '</strong> ' + property.value + '</li>');
-              if (property.name === 'fault') {
+              if (property.name === 'fault' || property.name === 'error') {
```

Comment:
Sorry, I don't understand what do you want to say. 

What I meant is that when someone is using `error` tag can accidentally pass other values, for instance `span.setTag(Tags.ERROR.getKey(), "whatever")`. In this case we can be more predictive and assume that span is in error state and highlight it.

...when using `Tags` api it does not allow to set wrong value: `Tags.ERROR.set(span,<< boolean>>)`